api/odl2.py

```diff
@@ -251,6 +258,7 @@ def _extract_publication_metadata(self, feed, publication, data_source_name):
                     if not medium:
                         medium = Edition.medium_from_media_type(license_format)
 
+                    drm_schemes: List[str | None]
```

Comment:
Do any of the calls in the following if/else block return a list that potentially includes `None`? Or does allowing `None1 here come in only for the `for` on l. 283?